src/games/strategy/engine/framework/startup/ui/EnginePreferences.java

```diff
@@ -372,6 +377,14 @@ private void setupListeners() {
         ProcessRunnerUtil.exec(commands);
 
     }));
+    m_mapXmlCreator.addActionListener(SwingAction.of("[Beta] Run the Map Creator", e -> {
+      final List<String> commands = new ArrayList<>();
```

Comment:
DRY violation.
Can use a helper method to return a Runnable that takes in 'javaClass' as a parameter